### PR TITLE
feat: add new translations for NL and JP

### PR DESCRIPTION
**Pull request by Karol-Stelmaczonek**
Ticket: https://oat-sa.atlassian.net/browse/AUT-2969

## What's Changed

- Add translations nl-NL and ja-JP for msgstr: 
  - Submit and go to the item %s
  - Submit and go to the next item
  - Submit and go to the previous item

### CODE REVIEW

File: locales/ja-JP/messages.po [hunk 1]
```diff
@@ -442,8 +442,10 @@ msgstr "エラー"
 msgid "Error during file import"
 msgstr "ファイルインポート中のエラー"
 
-msgid "Error found in the IMS QTI Test:\n%s"
-msgstr "QTIテストにエラーが見つかりました:\n%s"
+msgid "Error found in the IMS QTI Test:\n"
+"%s"
```
Comment (bziondik) [inline locales/ja-JP/messages.po:446]: This is strange
please, check this key in code

File: locales/ja-JP/messages.po [hunk 2]
```diff
@@ -788,12 +787,15 @@ msgstr "このテストパートの最小時間"
 msgid "minutes"
 msgstr "分"
 
-msgid "Missed required parameter \'id\'"
-msgstr "必要なパラメータが不足しています \'id\'"
+msgid "Missed required parameter \'id\"
+msgstr ""
```
Comment (bziondik) [inline locales/ja-JP/messages.po:791]: Check this also, and copy translations

File: locales/ja-JP/messages.po [hunk 3]
```diff
@@ -1428,7 +1427,12 @@ msgstr "セクション名"
 msgid "The submission mode determines when the candidate's responses are submitted for response processing. A testPart in individual mode requires the candidate to submit their responses on an item-by-item basis. In simultaneous mode the candidate's responses are all submitted together at the end of the testPart."
 msgstr "受験者の解答がどのようなタイミングで送信されるかを設定します。「個別」送信モードのテストパートでは、受験者はアイテムごとに解答を提出する必要があります。「同時」送信モードでは、受験者の解答はテストパートの最後にまとめて送信されます。"
 
-msgid "The test cannot be saved because it currently contains invalid settings.\nPlease fix the invalid settings and try again."
+msgid "The test cannot be saved because it currently contains invalid settings.\n"
```
Comment (bziondik) [inline locales/ja-JP/messages.po:1430]: This also looks like broken

File: locales/ja-JP/messages.po [hunk 4]
```diff
@@ -1428,7 +1427,12 @@ msgstr "セクション名"
...
+""
+msgstr ""
+
+msgid "The test cannot be saved because it currently contains invalid settings.\n"
+"Please fix the invalid settings and try again."
```
Comment (bziondik) [inline locales/ja-JP/messages.po:1435]: ?

File: locales/ja-JP/messages.po (hunk 1 above)
Comment (Karol-Stelmaczonek) [reply locales/ja-JP/messages.po:446]: Its here: taoQtiTest/models/classes/class.QtiTestService.php:754 I looked in code and there's other *.po files not touched by me with such string like here: taoQtiTest/locales/da-DK/messages.po:427

**Review by bziondik — APPROVED** [review]
- [ ]  New code is covered by tests (if applicable)
- [ ]  Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x]  New code is respecting code style rules
- [x]  New code is respecting best practices
- [x]  New code is not subject to concurrency issues (if applicable)
- [ ]  Feature is working correctly on my local machine (if applicable)
- [x]  Acceptance criteria are respected
- [x]  Pull request title and description are meaningful

File: locales/ja-JP/messages.po (hunk 2 above)
Comment (Karol-Stelmaczonek) [reply locales/ja-JP/messages.po]: Reverted

**Review by andreluizmachado — COMMENTED** [review]
[ ] New code is covered by tests (if applicable)
[ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
[x] New code is respecting code style rules
[x] New code is respecting best practices
[x] New code is not subject to concurrency issues (if applicable)
[ ] Feature is working correctly on my local machine (if applicable)
[x] Acceptance criteria are respected
[x] Pull request title and description are meaningful

**Review by andreluizmachado — APPROVED** [review]

**Pull request merged**